### PR TITLE
Documentation for new Transport NSW sensor

### PR DESCRIPTION
**Pull request by Dav0815**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#17242

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/sensor.transport_nsw.markdown [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+# Example configuration.yaml entry
+sensor:
+  - platform: transport_nsw
+    name: 'Bus E80'
+    stopid: '200024'
+    route: 'E80'
+    apikey: 'YOUR API KEY'
+```
+
+Configuration variables:
```
Comment (WofWca) [inline source/_components/sensor.transport_nsw.markdown]: [This paragraph](https://developers.home-assistant.io/docs/en/documentation_create_page.html#configuration) of the standard is not followed.

File: source/_components/sensor.transport_nsw.markdown [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+You can define a bus line, but if you don’t do it, the sensor will pick up the next stop event from any line servicing this stop.
+
+Then add the data to your `configuration.yaml` file as shown in the example:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: transport_nsw
+    name: 'Bus E80'
```
Comment (WofWca) [inline source/_components/sensor.transport_nsw.markdown]: > [This sample must contain only the required variables](https://developers.home-assistant.io/docs/en/documentation_create_page.html#configuration)

**Review by WofWca — CHANGES_REQUESTED** [review]

File: source/_components/sensor.transport_nsw.markdown (hunk 2 above)
Comment (frenck) [reply source/_components/sensor.transport_nsw.markdown]: Correct, like @WofWca reports.

We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

File: source/_components/sensor.transport_nsw.markdown (hunk 1 above)
Comment (frenck) [reply source/_components/sensor.transport_nsw.markdown]: Correct, like @WofWca reports.

Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration

File: source/_components/sensor.transport_nsw.markdown [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+title: "Transport NSW"
+description: "Instructions on how to integrate timetable data for Transport NSW (Australia) within Home Assistant."
+date: 2018-08-18 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Transport
+ha_iot_class: "Cloud Polling"
+ha_release: 0.76
```
Comment (frenck) [inline source/_components/sensor.transport_nsw.markdown]: This release has been shipped quite some time ago.
Please update this to the next major release.

**Review by frenck — CHANGES_REQUESTED** [review]
@Dav0815 Thank you for this PR. @WofWca left some comments. Could you please take a look? 👍

File: source/_components/sensor.transport_nsw.markdown (hunk 1 above)
Comment (Dav0815) [reply source/_components/sensor.transport_nsw.markdown]: Should be fixed now.

**Review by WofWca — DISMISSED** [review]

**Review by frenck — DISMISSED** [review]

**Review by frenck — DISMISSED** [review]

**Comment by frenck** [comment]
Thanks, @Dav0815 for the updates!

✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by Dav0815** [comment]
@frenck how/where do I add the logo for this component ?

**Comment by frenck** [comment]
@Dav0815 -> `source/images/supported_brands/` 🖼

**Review by frenck — DISMISSED** [review]

**Comment by frenck** [comment]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

File: source/_components/sensor.transport_nsw.markdown [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+description: "Instructions on how to integrate timetable data for Transport NSW (Australia) within Home Assistant."
+date: 2018-10-05 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: transport_nsw.png
+ha_category: Transport
+ha_iot_class: "Cloud Polling"
+ha_release: "0.80"
```
Comment (MartinHjelmare) [inline source/_components/sensor.transport_nsw.markdown]: 0.81

**Review by frenck — APPROVED** [review]

**Pull request merged**